examples/seismic/tti/tti_example.py

```diff
@@ -41,6 +41,7 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
         warning('WARNING: TTI requires a space_order that is a multiple of 4!')
 
     rec, u, v, summary = solver.forward(autotune=autotune, kernel=kernel)
+    rec1, u1, v1, summary1 = solver.forward_staggered(autotune=autotune, kernel=kernel)
```

Comment:
I don't think we want to execute both back to back -- but I suspect this is just a leftover of the work you're doing